### PR TITLE
Release 1.1.7

### PR DESCRIPTION
**Pull request by thiemowmde**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,10 @@ It is based upon and contains a lot of code written by [Jeroen De Dauw]
 
 ## Release notes
 
+### 1.1.7 (2016-05-25)
+
+* Fixed `FORMAT_NAME` string constants in all parsers
```
Comment (JeroenDeDauw) [inline README.md]: This implies they where "broken".

Comment (thiemowmde) [inline README.md]: If it's not a fix, what is it then? A breaking change?

Comment (JeroenDeDauw) [inline README.md]: Refactoring? Anyway, regardless of the precise classification, I'd not mention it in the release notes, as I don't see how it is of interest to users of this component. And if it is, then one needs to provide the info they need.

Comment (thiemowmde) [inline README.md]: Ok, but what should the release notes say then? I would like to do a patch-level or minor release, but what from https://github.com/DataValues/Geo/compare/1.1.6...master is worth mentioning?

Comment (JeroenDeDauw) [inline README.md]: How about "Made minor documentation improvements"?

Comment (thiemowmde) [inline README.md]: Works. Thanks!

**Comment by JeroenDeDauw** [comment]
@thiemowmde you making the tag?

**Pull request merged**